GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -1007,30 +1008,23 @@ private static GitItemStatus GitItemStatusFromStatusCharacter(StagedStatus stage
             };
         }
 
-        public static ArgumentString MergeBranchCmd(string branch, bool allowFastForward, bool squash, bool noCommit, string strategy, bool allowUnrelatedHistories, string message, int? log)
+        public static ArgumentString MergeBranchCmd(string branch, bool allowFastForward, bool squash, bool noCommit, string strategy, bool allowUnrelatedHistories, string folderPath, int? log)
```

Comment:
Please add tests for this method that at least cover the `-F` flag

Reply:
All the test cases now accept a path instead of a message.

The response has `-F` flag.

https://github.com/gitextensions/gitextensions/blob/5935979af57db41db5a4e128e2d058422fb9bb57/UnitTests/GitCommands.Tests/Git/GitCommandHelpersTest.cs#L747